joomla/joomla-cms
# Number validation as global handler

Pull Request for Issue #5619 .
#### Summary of Changes
- Added handler for the number field validation
- Minified uncompressed file
#### Testing Instructions
1. Go to the Joomla administration
2. Then: System => Global Configuration => System (Tab) => Session Settings 
3. Try input in the field 'Session Lifetime' zero or negative value

You should see error (red border around and title on mouseover)

This PR add handler as global, only for session lifetime (com_config) here: #9997
